.bazelrc

```diff
@@ -1,9 +1,25 @@
+common --registry=https://bcr.bazel.build
+common --extra_toolchains=@llvm_17_toolchain//:all
+
+common --@toolchains_llvm//toolchain/config:libunwind=False
+build --linkopt --unwindlib=libgcc
+
+common:clang-18 --extra_toolchains=@llvm_18_toolchain//:all
+
+build:system-clang --extra_toolchains=@bazel_tools//tools/cpp:toolchain
+build:system-clang --action_env=BAZEL_COMPILER=clang
+build:system-clang --cxxopt=-std=c++20 --host_cxxopt=-std=c++20
+build:system-clang --linkopt -fuse-ld=lld
 # use a compiler name that doesn't symlink to ccache
-build --action_env=CC=clang-16 --host_action_env=CC=clang-16
-build --action_env=CXX=clang++-16 --host_action_env=CXX=clang++-16
-build --action_env=BAZEL_COMPILER=clang
-build --cxxopt=-std=c++20 --host_cxxopt=-std=c++20
-build --linkopt -fuse-ld=lld
+build:system-clang-16 --config=system-clang
```

Comment:
because `common --extra_toolchains=@llvm_17_toolchain//:all` is common, i assume it means it automatically composes with other options. so i'm curious what it means to have llvm17 toolchain and system-clang-16 selected.

Reply:
The last `--extra_toolchains` is preferred. So when you pick `--system-clang-16` it should override the llvm toolchain with the bazel default toolchain that will respect CC.

Reply:
Docs if you're interested: https://bazel.build/extending/toolchains#toolchain-resolution

Reply:
ahh i didn't see `--config=system-clang` sitting at the end. cool